src/TestLogger/Extensions/XunitTestAdapter.cs

```diff
@@ -46,6 +46,14 @@ public class XunitTestAdapter : ITestAdapter
                     result.Messages.Add(new TestResultMessage("skipReason", skipReason));
                 }
 
+                string displayName = result.Result.DisplayName;
+
+                // Add parameters for theories.
+                if (displayName is string dn && dn.Contains("("))
```

Comment:
This works, but would fail if classes could be passed parameters in xunit. I couldn't find a way to do that, so i'm just accepting this. as is.